Add LiteDBStore (#269)
planetarium/libplanet: CHANGES.md

### Added interfaces

>  -  Added `LiteDBStore` that using [LiteDB] as backend. [[#269]]

Reviewer: ```suggestion
 -  Added `LiteDBStore` backend that uses [LiteDB] under the hood.  [[#269]]
```
